.github/workflows/pipeline.yml

```diff
@@ -1,14 +1,14 @@
 name: Pipeline
-on: [pull_request]
+on: [push, pull_request]
```

Comment:
Not a fan of pipeline running on prs and pushes as it duplicates but not sure there's anything we can do about it now and it's nice to see results when we push to a new branch (before creating the PR)

Reply:
Yeah, I agree.